azure-pipelines.yml

```diff
@@ -1,4 +1,16 @@
 jobs:
+- job: BiDiScan
+  pool:
+    vmImage: 'ubuntu-18.04'
+
+  steps:
+  - checkout: self
+    fetchDepth: 32
+
+  - script: |
+      git fetch origin 5.7
+      python $(Build.SourcesDirectory)/scripts/find_unicode_control.py -p bidi -v $(git diff --name-only --relative --diff-filter AMR origin/5.7 -- . | tr '\n' ' ')
```

Comment:
It checks also e.g. `.test` and `.result` files. You may limit checking to some types of files e.g.: 
```git diff --name-only origin/5.7 *.c *.cc *.cpp *.h *.hpp *.i *.ic *.ih```

Reply:
@inikep, maybe better to exclude ones which we are sure we don't want to check (like .test and .result)?

Reply:
Sure, it will be better. For now you can keep all files and improve it later.